### PR TITLE
Fix Bundle #2: Fixed Sidenav Inner Children rendering issues, Link content not inlined, Breadcrumbs fix for missing index directories

### PR DESCRIPTION
**Pull request by MaximumCrash**
### Attention (CC)
@twblack88 
@andytudhope 
@isaacpearl 

## Summary of changes
- Fixed Sidenav children that didn't have index pages from not being rendered, causing the sidenav to have missing children. (ie. The case where /work_with_us/governance/take-notes <-- THIS Doesn't have an index file, but it has children)
- Added chevrons to sidenav nodes that have children, this wasn't in the spec, but it's obvious that it's needed. 
- Fixed Link inner content not being inlined (sometimes copy would be rendered as a div and divs are block elements by default)
- Fixed breadcrumbs having similar missing index issue as the sidenav. 

## Motivation & Context
- Issue #82, #83 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

## How Has This Been Tested?
Locally

### CODE REVIEW

**Comment by MaximumCrash** [comment]
@all-contributers please add @MaximumCrash for code

File: README.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ _See the official [Dai-UI](https://github.com/makerdao/dai-ui) repo for more in-
     ```
 
 2.  **Install**
- _dai-ui-theme-maker and dai-ui-icons are pre-installed._
-  *This project makes use of `yarn`. You'll likely need to use npm to get everything setup once cloned.*
+    _dai-ui-theme-maker and dai-ui-icons are pre-installed._
+    _This project makes use of `yarn`. You'll likely need to use npm to get everything setup once cloned._
```
Comment (twblack88) [inline README.md:30]: _This project makes use of `yarn`. Don't forget to run `yarn` once before starting the environment up.

**Review by twblack88 — CHANGES_REQUESTED** [review]
everything is super clear and looks up to snuff. I wrote out one change to the readme that, hopefully, adds some clarity. We'll keep tinkering on that tho.

**Review by isaacpearl — APPROVED** [review]

**Comment by MaximumCrash** [comment]
Other PRs that started from this PR's branch have been merged in prior. No need to deal with conflicts.

**Pull request closed without merging**